swarm/api/config.go

```diff
@@ -107,7 +109,7 @@ func (self *Config) Init(prvKey *ecdsa.PrivateKey) {
 	self.BzzKey = keyhex
 
 	self.Swap.Init(self.Contract, prvKey)
-	self.SyncParams.Init(self.Path)
-	self.HiveParams.Init(self.Path)
+	//self.SyncParams.Init(self.Path)
+	//self.HiveParams.Init(self.Path)
```

Comment:
why is `self.HiveParams.Init()` also commented if they are not commented out like ``self.SyncParams`?

Reply:
I'm not sure I understand the question, but `network.HiveParams.Init()` doesn't exist anymore. Maybe the line should be deleted.

Reply:
`network.HiveParams.Init()` is in the current `master`. This is potentially a problem then